frontend/containers/DocumentList/DocumentList.jsx

```diff
@@ -168,6 +195,28 @@ class DocumentList extends Component {
     actions.clearDocumentList()
   }
 
+  renderField(fieldName, schema, value) {
```

Comment:
That will probably involve another endpoint which the front-end will need to ping in order to know which field components are available.